pyalarmdotcomajax/pyalarmdotcomajax.py

```diff
@@ -89,97 +98,110 @@ def __init__(
         }
         self._systemid = None
         self._partitionid = None
+        self._lockid = None
         self._forcebypass = forcebypass  # "stay","away","true","false"
         self._noentrydelay = noentrydelay  # "stay","away","true","false"
         self._silentarming = silentarming  # "stay","away","true","false"
         self._thermostat_detected = False
+        self._lock_detected = False
         self._garage_door_detected = False
         self._url_base = self.URL_BASE
         self._twofactor_cookie = (
             {"twoFactorAuthenticationId": twofactorcookie} if twofactorcookie else {}
         )
 
-    async def _async_get_ajax_key(self):

```

Comment:
Any idea why the diff here is so weird? Maybe needs a rebase?

Reply:
No, seems some whitespace changes snuck in. If you, when you look at the diff, push the wheel and select "hide whitespace", you will not see it.

Feel free to ask me to update the branch with one commit to fix this, if you don't like the commit :)

Reply:
I see, yes, can you reformat the whitespace? Just use black with the default settings.

Reply:
I actually have no idea what whitespace it complains about.

If you view the diff with "hide whitespace" mode, you'll not see these changes (they're not really changes).

Reply:
Resolved now